.github/workflows/update-ssdlc-report.yaml

```diff
@@ -0,0 +1,118 @@
+name: Update Compliance Report
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'atlascli tag version (e.g. 1.42.2)'
+        required: true
+        type: string
+
+jobs:
+  update-compliance-report:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GitHubSecurityLab/actions-permissions/monitor@v1
+        with:
+          config: ${{ vars.PERMISSIONS_CONFIG }}
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Extract AUTHOR and VERSION
+        id: extract
+        run: |
+          if [[ "${GITHUB_EVENT_NAME}" == "workflow_dispatch" ]]; then
+            # Expect input like 'atlascli/v1.42.2' or '1.42.2'
+            TAG="${{ github.event.inputs.tag }}"
+            VERSION="${TAG#atlascli/v}"
+            AUTHOR="${{ github.actor }}"
+          else
+            # GITHUB_REF is refs/tags/atlascli/v1.42.2
+            VERSION="${GITHUB_REF#refs/tags/atlascli/v}"
+            AUTHOR="${{ github.event.release.author.login }}"
+          fi
+          echo "author=$AUTHOR" >> "$GITHUB_OUTPUT"
+          echo "version=$VERSION" >> "$GITHUB_OUTPUT"
+      - name: Run gen-ssdlc-report.sh
+        env:
+          AUTHOR: ${{ steps.extract.outputs.author }}
+          VERSION: ${{ steps.extract.outputs.version }}
+        run: ./build/package/gen-ssdlc-report.sh
```

Comment:
Is there a risk that gen-ssdlc-report.sh run on a codebase that no longer reflect the tag? What I mean here is that I don't see any step where you are getting the CLI codebase at the release tags so the ssdlc report may includes code changes that were merged after the release tag was created.

[solution] I think you should update the checkout step to download the codebase at the release tag

Reply:
> includes code changes that were merged after the release tag was created.

good point, but since the report simply links to the tag link I think it will always show the sbom report that was generated upon tag release, right?

Reply:
ah, I don't know what `gen-ssdlc-report.sh` does 😅 Feel free to ignore the comment if the script does not actually do anything related to the codebase.

Reply:
not for now, but its a good point! thanks